webui/react/src/pages/TrialsComparison/Collections/useTrialCollections.tsx

```diff
@@ -52,23 +50,30 @@ const getDefaultFilters = (projectId: string) => (
   { projectIds: [ String(projectId) ] }
 );
 
+const defaultSorter: TrialSorter = {
+  sortDesc: true,
+  sortKey: 'trialId',
+};
+
 export const useTrialCollections = (
   projectId: string,
-  tableSettings: InteractiveTableSettings,
+  tableSettingsHook: SettingsHook<InteractiveTableSettings>,
 ): TrialsCollectionInterface => {
+  const { settings: tableSettings, updateSettings: updateTableSettings } = tableSettingsHook;
   const filterStorage = useStorage(`trial-filters}/${projectId ?? 1}`);
   const initFilters = filterStorage.getWithDefault<TrialFilters>(
     'filters',
     getDefaultFilters(projectId),
   );
 
-  const [ sorter, setSorter ] = useState<TrialSorter>({
-    orderBy: tableSettings.sortDesc ? V1OrderBy.DESC : V1OrderBy.ASC,
-    sortKey: tableSettings.sortKey ? String(tableSettings.sortKey) : 'trialId',
-  });
-
```

Comment:
this was basically just duplicating the sorter state from the table. instead now we just use it directly